test/unit/quality-labels-test.js

```diff
@@ -0,0 +1,99 @@
+define([
+    'utils/quality-labels',
+    'utils/underscore',
+], function (qualityLabels, _) {
+    QUnit.module('quality-labels');
+    var test = QUnit.test.bind(QUnit);
+
+    var customLabels  = {
+        1000: 'low',
+        2000: 'medium',
+        3000: 'high',
+    };
+
+    var bandwidths = _.keys(customLabels);
+
```

Comment:
Can you add `Qunit.module` before each method thats being tested? Its a useful separator, just like how we would use `describe` in mocha.